pages/children.md

```diff
@@ -10,10 +10,9 @@ Our day for children is back!
 We're hosting a fun-packed day of activities and workshops for children to
 learn all about getting the most out of the world with Python.
 
-We're working with the Raspberry Pi Foundation, and some of the team who
-brought Python to the Micro:bit will be coming along to show you how use Python
-to write games.  There'll be a chance to play with robots, and to learn how to
-program Minecraft with Python!
+This year we're working with the Raspberry Pi Foundation, Code Club and some of the team who
+brought Python to the BBC Micro:bit. They will be coming along to show you how use Python
```

Comment:
Unfortunately I don't have write access to the repo (yet), so I can't do anything directly here.

Happy to do so if somebody grants me write access (I know @inglesp mentioned this yesterday?).